x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_details/tabs/tab_summary.tsx

```diff
@@ -130,6 +132,22 @@ export const TabSummary: React.FunctionComponent<Props> = ({ policy }) => {
 
   return (
     <Fragment>
+      {isManagedPolicy ? (
+        <>
```

Comment:
Looks like we are already inside a fragment (L134) so can remove this.

Reply:
I think this is still needed here, as we're also rendering `EuiSpacer`